admin-notice/class-admin-notice.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Automattic\VIP\Admin_Notice;
+
+class Admin_Notice {
+	public $message;
+	public $start_date;
+	public $end_date;
+
+	/**
+	 * Create AdminNotice
+	 *
+	 * $message - the text to be displayed
+	 * $start_date - the date* since the notice should start to show up
+	 * $end_date - the date* till the notice should show up
```

Comment:
Use proper `@param` tags.

Reply:
I updated it. From what I googled this should be now the correct proper form, but feel free to point out more specifically what is wrong (this is the first time I am using those tags).

Reply:
Have a read through https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#docblock-formatting for how it should be in WP, and https://docs.phpdoc.org/3.0/guide/references/phpdoc/tags/param.html#param (and maybe the [draft PSR-19](https://github.com/phpDocumentor/fig-standards/blob/master/proposed/phpdoc-tags.md#59-param)) for general usage.